### PR TITLE
Add favicons

### PR DESCRIPTION
**Pull request by tadhg-ohiggins**
Show icons to all.
Under the tabs, Mario
Jumps but falls just short.

Adds the assets for favicon images, and the requisite code for them.

### CODE REVIEW

File: ui/components/html.jsx [hunk 1]
```diff
@@ -35,6 +35,58 @@ function newRelicTag() {
...
+      name="msapplication-TileImage"
+      content="mstile-144x144.png"
+    />,
+    msSquares,
+    <meta
+      name="msapplication-wide310x150logo"
+      content="mstile-310x150.png"
+    />,
+  ]);
+}
```
Comment (cmc333333) [inline ui/components/html.jsx]: NBD, but React might complain about wanting specific `key` attributes for each element as a rendering optimization. Easy enough to use their index (though you'll need to temporarily disable the linter check for that).

File: ui/components/html.jsx [hunk 2]
```diff
@@ -35,6 +35,58 @@ function newRelicTag() {
...
+  );
+
+  return ([
+    appleIcons,
+    icons,
+    <meta name="application-name" content="&nbsp;" />,
+    <meta name="msapplication-TileColor" content="#FFFFFF" />,
+    <meta
+      name="msapplication-TileImage"
+      content="mstile-144x144.png"
```
Comment (cmc333333) [inline ui/components/html.jsx]: Does this need to have the `/static/img/favicon/` prefix?

File: ui/components/html.jsx [hunk 3]
```diff
@@ -35,6 +35,58 @@ function newRelicTag() {
...
+    />,
+    msSquares,
+    <meta
+      name="msapplication-wide310x150logo"
+      content="mstile-310x150.png"
```
Comment (cmc333333) [inline ui/components/html.jsx]: Same prefix question here.

File: ui/components/html.jsx (hunk 2 above)
Comment (tadhg-ohiggins) [reply ui/components/html.jsx]: These both do, thanks for catching that!

**Pull request merged**